### PR TITLE
command position method created

### PR DESCRIPTION
**Pull request by IlyaStrelyabin**

### CODE REVIEW

File: data/match_results.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+  "match_2": {
```
Comment (Major666RUS) [inline data/match_results.json:2]: It seems to me that it doesn't meet our data structure agreement. I mean using an array instead of an object. Also I have one more question. Should we "initialize" our file with empty values? Or is it just example?

File: data/schedule.json [hunk 2]
```diff
@@ -1,434 +0,0 @@
-[
```
Comment (Major666RUS) [inline data/schedule.json]: What's the reason for removing this file?

File: README.md [hunk 3]
```diff
@@ -1,11 +1 @@
...
-Overview
---------
-
-This is an amazing football simulator inspired by outcoming FIFA World Cup 2022 in Qatar. Select a team, simulate real matches and win the cherished trophy!
-
-Demo
---------
-Link will be here soon.
-
+# cotletico-manager
```
Comment (Major666RUS) [inline README.md]: Could you revert "correct" version of README.md

File: data/group_stage.json [hunk 4]
```diff
@@ -0,0 +1,338 @@
+{
```
Comment (Major666RUS) [inline data/group_stage.json:1]: I would suggest to use more informative keys in the structure like wins, draws, losses etc

File: data/group.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "A": ["Qatar", "Ecuador", "Senegal", "Netherlands" ],
-  "B": ["England", "Iran", "United_States", "Unknown"],
+  "B": ["England", "Iran", "USA", "Unknown"],
   "C": ["Argentina", "Saudi_Arabia", "Mexico", "Poland"],
   "D": ["France", "Unknown", "Denmark", "Tunisia"],
```
Comment (iibykov) [inline data/group.json]: We have 3 teams with the same name **Unknown**.
It is proposed to rename them to Unknown_**B**, Unknown_**D** and Unknown_**E**.
National teams should have similar names in the _schedule.json_ file. It needs to be corrected.

File: data/group_stage.json [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 0
+    },
+    "United_States": {
```
Comment (iibykov) [inline data/group_stage.json]: Replace with **USA**

File: data/group_stage.json [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+      "pld": 0,
+      "w": 0,
+      "d": 0,
+      "l": 0,
+      "gf": 0,
+      "ga": 0,
+      "gd": 0,
+      "p": 0
+    },
+    "Unknown": {
```
Comment (iibykov) [inline data/group_stage.json]: Replace **Unknown** teams (Unknown_B, Unknown_D and Unknown_E) as per comment above.

File: src/cotletico_manager.py [hunk 8]
```diff
@@ -5,24 +5,6 @@
 import match
 
 
-def schedule_from_json(filename: str):
```
Comment (iibykov) [inline src/cotletico_manager.py]: The **_schedule_from_json_** method has been removed from _cotletico_manager.py_ but not moved to _schedule.py_.

Move to _schedule.py_ to schedule class and rename method to **_from_json_file_**.

File: src/cotletico_manager.py [hunk 9]
```diff
@@ -5,24 +5,6 @@
...
-        matches.append(match)
-
-    schedule = Schedule(matches)
-    return schedule
-
-
-schedule = schedule_from_json('../data/schedule.json')
-
-
 def teams_from_json(filename: str):
```
Comment (iibykov) [inline src/cotletico_manager.py]: Move to _team.py_ to _team_ class and rename method to **_from_json_file_**.

File: src/group.py [hunk 10]
```diff
@@ -7,6 +8,11 @@ def __init__(self, name, teams):
         self.name = name
         self.teams = teams
 
+    @staticmethod
+    def team_position(team_name):
+        """shows the team's current position in the group"""
+        return team_position.update_status(team_position.group_name(team_name))
+
 
 def groups_from_json(filename: str):
```
Comment (iibykov) [inline src/group.py]: The method returns nothing. Most likely we wanted to return **_groups_**.

File: src/team_position.py [hunk 11]
```diff
@@ -0,0 +1,396 @@
...
+    groups = json.load(file)
+
+with open('../data/match_results.json') as file:
+    match_results = json.load(file)
+
+with open('../data/group.json') as file:
+    g_name = json.load(file)
+
+
+class Match:
```
Comment (iibykov) [inline src/team_position.py:17]: Why do we need another Match class?

File: src/team_position.py [hunk 12]
```diff
@@ -0,0 +1,396 @@
+import json
```
Comment (iibykov) [inline src/team_position.py:1]: It seems like it could be done a lot easier. I'll think about it.

**Review by iibykov — CHANGES_REQUESTED** [review]
Thank you for your work, I left some comments on your request. Please take a look.

File: data/match_results.json (hunk 1 above)
Comment (IlyaStrelyabin) [reply data/match_results.json:2]: You are right, this is just an example

File: src/cotletico_manager.py (hunk 9 above)
Comment (IlyaStrelyabin) [reply src/cotletico_manager.py]: Added a method from the match_generator branch

File: src/cotletico_manager.py (hunk 8 above)
Comment (IlyaStrelyabin) [reply src/cotletico_manager.py]: Added a method from the match_generator branch

File: src/team_position.py (hunk 11 above)
Comment (IlyaStrelyabin) [reply src/team_position.py:17]: It was added to run the code in test mode

File: src/schedule.py [hunk 13]
```diff
@@ -8,3 +15,146 @@ def add_match(self, match):
...
+            if gest_name == t.name:
+                gest_team = t
+                gest_rating = t.get_rating(m, teams)
+                print(f'Стартовый состав команды  гостей {gest_name}:')
+                t.first11_presentation(m, teams)
+
+        print(f'рейтинг команд: {host_name} - {host_rating}   :    {gest_name} - {gest_rating}')
+        print()
+
+        chance = ev.ChanceState()
```
Comment (Major666RUS) [inline src/schedule.py:64]: It seems to be irrelevant here because class Event doesn't have such methods. I would be better to do git merge from develop to see current the develop branch.

File: src/team.py [hunk 14]
```diff
@@ -2,6 +2,8 @@
 import stadium
 import event
```
Comment (Major666RUS) [inline src/team.py:3]: unnessary imports should be deleted

**Review by Major666RUS — COMMENTED** [review]
Good job!
